book_source/02_demos_tutorials_workflows/02_user_demos/06_Workflow_using_Rmd_template.Rmd/workflow.Rmd

```diff
@@ -0,0 +1,124 @@
+# Introduction
+
+This tutorial provides a step-by-step guide to running the **PEcAn Modular Workflow**. The PEcAn (Predictive Ecosystem Analyzer) system automates ecological modeling, helping researchers analyze plant functional traits, run model simulations, and perform sensitivity analyses.
+
+After setting up PEcAn locally, open **RStudio** in your browser and log in with:
+- **Username**: `carya`
+- **Password**: `illinois`
+
+Then, navigate to `web/workflow.Rmd` and open it to run the workflow. This method eliminates the need to execute the workflow through the web interface once `pecan.xml` is configured.
+
+---
+
+# Prerequisites
+
+Before running this workflow, ensure you have:
+
+- Installed **PEcAn** and its dependencies.
+- An XML settings file (`settings.xml`) configured for your use case.
+- A model binary (e.g., **SIPNET** or **ED2**) specified in your settings.
+
+---
+
+# Load PEcAn Packages
+
+We first load the necessary PEcAn modules required for running the workflow.
+
+```{r libraries, message=FALSE, warning=FALSE}
+library("PEcAn.all")
+```
+
+---
+
+# Load PEcAn Settings
+
+We read the PEcAn settings from an XML file. This file contains configuration details such as input datasets, model selection, and output directories.To create a pecan.xml, you can download one generated in the PEcAn web interface or one of the `pecan.<modelname>.xml` files in the tests/ directory of 
+the PEcAn repository (github.com/pecanproject/pecan).
+
+```{r load-settings}
+settings_path <- "settings.xml"  # Path to the settings file
+settings <- PEcAn.settings::read.settings(settings_path)
+```
+
+---
+
+# Prepare and Validate Settings
+
+PEcAn provides utilities to process and validate settings before execution. This step ensures that required fields are properly set.
+
+```{r prepare-settings}
+settings$outdir <- "/home/carya/pecan_runs"
```

Comment:
This line isn't being explained but is important.  Also, if outdir needs to be updated before prepare.settings this needs to be explained.